week-3-practise/exercise3.js

```diff
@@ -0,0 +1,10 @@
+// find all the names thats start with an A :
+
+var names = ["Alice", "benjamin", "Angela", "Adam", "Bob"];
+// write answer here
+var nameStartWithA = names.filter(name => name[0] === "A");
```

Comment:
This is technically correct, but I think the *intention* of the question is a little different.

What happens if I change "Alice" to "alice"? Is it still correct? How could we fix it?